domains/POAS/events/notification-sent/schema.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$id": "https://opg.service.justice.gov.uk/opg.poas.sirius/notification-sent.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "opg.poas.makeregister/notification-sent",
+  "type": "object",
+  "properties": {
+    "uid": {
+      "type": "string",
+      "description": "The UID of the LPA",
+      "pattern": "M(-[A-Z0-9]{4}){3}"
+    },
+    "deliveryMethod": {
+      "type": "string",
+      "description": "How the notification was sent",
+      "enum": ["Email", "SMS"]
+    },
+    "subject": {
+      "type": "string",
+      "description": "What the notification was about",
+      "enum": ["CertificateProviderInvite"]
+    }
```

Comment:
It feels like this would create a tight couple between MRLPA, event store, and Sirius. Probably easier to just let this be a free-text string, or omit it if it's not going to be used in Sirius.

It would also be good to provide the UUID of the sent notification from Notify, so that Sirius can retrieve any further details.

Reply:
I should have added the context on not being able to rely on UUIDs for what the email is about here rather than slack. Here it is:

> I was considering using the notify template ID as the identifier but theres no guarantee that the template ID will be static as our notify workflow involves creating new templates when personalisations change. Once we get to tickets that require the notification content we can look at where is best to do that

My understanding of the need behind this ticket was that a caseworker needs to be able to confirm an email (or SMS for future tickets) has been sent so they would need to know what the notification is about. We could pivot to having an event per notification (e.g. `certificate-provider-email-invite-sent`) rather than a general one but that's going to grow quite quickly from looking at our backlog (which may not necessarily be a bad thing 🤷 ).

For now, the free string is the easiest approach until handling the event and adding to timeline gets implemented on the Sirius side.

Reply:
Sorry I didn't meant the template ID, but the one for the specific notification. That gives Sirius freedom to fetch whatever it needs from the Notify API, rather than having to specify it in the event.

So Sirius might fetch the sent date, status and email content, and save that to a "Correspondence" DB record that it can show in the UI.